xcodeproj/internal/bazel_integration_files/copy_outputs.sh

```diff
@@ -19,6 +19,23 @@ else
       suffix=/Payload
     fi
 
+    if [[ -n ${BAZEL_OUTPUTS_DSYM:-} ]]; then
+      if [[ "$BAZEL_OUTPUTS_DSYM" = *.dSYM ]]; then
```

Comment:
Do we need this check? We have it above to allow for it to be zipped first, in which case we have to unzip it. Is it possible that it's zipped as well?

Reply:
No, I will remove it.